test/functional/cmdLineTests/criu/criu_jitserverPostRestore.xml

```diff
@@ -117,4 +121,83 @@
 		<output type="failure" caseSensitive="yes" regex="no">JITSERVER NO LONGER EXISTS</output>
 		<output type="failure" caseSensitive="yes" regex="no">User requested Java dump using</output>
 	</test>
+
+	<test id="Test SSL Success Case">
+		<command>bash $SCRIPPATH$ $TEST_RESROOT$ $TEST_JDK_BIN$ "$JVM_OPTIONS$" $MAINCLASS_OPTIONSFILE_TEST$ "JitOptionsTest $ENABLE_JITSERVER$ $JITSERVER_SSL1$ $SSL_VERBOSE$1" 1 false true</command>
+		<output type="success" caseSensitive="no" regex="no">Killed</output>
+		<output type="required" caseSensitive="yes" regex="no">Pre-checkpoint</output>
+		<output type="success" caseSensitive="yes" regex="no">Post-checkpoint</output>
+		<output type="failure" caseSensitive="yes" regex="no">CRIU is not enabled</output>
+		<output type="failure" caseSensitive="yes" regex="no">Operation not permitted</output>
+		<output type="required" caseSensitive="yes" regex="no">Successfully initialized SSL context</output>
+		<!-- If CRIU can't acquire the original thread IDs, this test will fail. Nothing can be done about this failure. -->
+		<output type="success" caseSensitive="yes" regex="no">Thread pid mismatch</output>
+		<output type="success" caseSensitive="yes" regex="no">do not match expected</output>
+		<output type="success" caseSensitive="yes" regex="no">Unable to create a thread:</output>
+		<output type="success" caseSensitive="yes" regex="no">JITSERVER EXISTS</output>
+		<output type="success" caseSensitive="yes" regex="no">JITSERVER STILL EXISTS</output>
+		<output type="failure" caseSensitive="yes" regex="no">JITSERVER DOES NOT EXIST</output>
+		<output type="failure" caseSensitive="yes" regex="no">JITSERVER NO LONGER EXISTS</output>
+	</test>
+
+	<test id="Check SSL Verbose Log for successful connection">
+		<command>bash $CATSCRIPPATH$ sslVlog1 false true</command>
```

Comment:
You should pass in `true true`; if a test fails at least you'll get the output of the previous test.

Reply:
> So the last bool set as true will not remove the vlogs as done [here](https://github.com/eclipse-openj9/openj9/blob/5e1ca2c503a6db436b2094debd8636fabb8d29c8/test/functional/cmdLineTests/criu/criuCatVlog.sh#L59C1-L59C1).

Setting the last bool to true **will** remove the vlogs. If it's set to false, then it will leave them alone.

> I understand the second last bool set as false would mean not to check the previous test output as used [here](https://github.com/eclipse-openj9/openj9/blob/5e1ca2c503a6db436b2094debd8636fabb8d29c8/test/functional/cmdLineTests/criu/criu_jitserverAcrossCheckpoint.xml#L53) for reading pre-checkpoint vlog and then set as true [here](https://github.com/eclipse-openj9/openj9/blob/5e1ca2c503a6db436b2094debd8636fabb8d29c8/test/functional/cmdLineTests/criu/criu_jitserverAcrossCheckpoint.xml#L60C1-L60C1) to read the test output log when reading post-restore vlog.

Right, but you're checking the post-restore vlog aren't you? There's no pre-checkpoint vlog specified in this test.

That's why I'm suggesting `true true`, because the first `true` will ensure we check the previous output in case the `"Test SSL Success Case"` fails for some unknown reason, and the second `true` to delete the vlogs, since you don't need it for the subsequent tests.

Reply:
@dsouzai Thanks understood. Will do the change.